util/network.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util // import "istio.io/fortio/util"
```

Comment:
in theory/for best practices we should avoid "util" as package name (https://blog.golang.org/package-names)

maybe `fnet` ? or just leave it in fhttp/ for now (though there is shared grpc and fhttp code I want to refactor so maybe `fnet` is not a bad idea)

Reply:
I'll change to fnet.